trieste/models/keras/sampler.py

```diff
@@ -121,12 +131,14 @@ def __init__(self, model: DeepEnsembleModel, diversify: bool):
 
         self._initialized = tf.Variable(False, trainable=False)
         self._batch_size = tf.Variable(0, dtype=tf.int32, trainable=False)
-        self._indices = tf.Variable(tf.zeros([0], dtype=tf.int32), shape=[None], trainable=False)
 
         if self._diversify:
-            self._sample_size = tf.Variable(0, dtype=tf.int32, trainable=False)
-            self._seeds = tf.Variable(
-                tf.zeros([0, 0], dtype=tf.float32), shape=[None, None], trainable=False
+            self._indices = tf.Variable(
```

Comment:
Why is this now called `_indices` if it's still a collection of floats? Maybe call it `_percentiles`?